rta/__init__.py

```diff
@@ -39,6 +45,116 @@ def get_ttp_names(os_types=None):
     return names
 
 
+def get_available_tests(print_list=False, os_filter=None):
+    test_names = []
+    test_metadata = []
+
+    for file in list(CURRENT_DIR_PATH.rglob("*.py")):
```

Comment:
```suggestion
    for file in CURRENT_DIR_PATH.rglob("*.py"):
```

also, do we want it to be recursive at this point?